_posts/2022-05-03-solidity-summit-2022-recap.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+published: true
+title: 'Solidity Summit 2022 Recap'
+date: '2022-05-03'
+author: Franziska Heintel
+category: Announcements
+---
+
+We can't believe it's already been two weeks since we met in Amsterdam for the second [Solidity Summit](https://summit.soliditylang.org/)!
+
+[Solidity Summit 2022](https://blog.soliditylang.org/2022/02/22/solidity-summit-2022-announcement/) was part of [Devconnect](https://devconnect.org/) and took place on Wednesday, April 20, 2022.
+
+With roughly 250 participants, the event was well attended. In addition, approximately 400 people joined remotely by watching the [Livepeer livestream](https://streameth.tv/event/solidity-summit).
+
+The day was packed with 20+ talks on
+- Solidity internals & deep dives
+- Solidity language design
+- Solidity tooling
+- Security
+- Programming patterns
+
+... and more.
+
+The full agenda of the day can be found [here](https://summit.soliditylang.org/agenda.html).
+
+Sorted by topic, here are all talks with links to the recordings and slides.
+
+### Solidity Deep Dives
+- Solidity in 2022: Recent and Planned Features - Christian Reitwiessner
+    - [Recording](https://youtu.be/V7_qTK5LIeA)
+    - [Slides](https://github.com/ethereum/solidity-summit/blob/master/2022/slides/1010_ChristianReitwiessner_Solidity%20in%202022%20Recent%20and%20Planned%20Features.pdf)
+- The Solidity Optimizooooor - Hari Mulackal
+    - [Recording](https://youtu.be/BWO7ij9sLuA)
+    - [Slides](https://github.com/ethereum/solidity-summit/blob/master/2022/slides/1640_HariMulackal_The%20Solidity%20Optimizooooor.pdf)
+- Generating EVM Bytecode from Yul in the New via-IR Pipeline - Daniel Kirchner
```

Comment:
```suggestion
- Generating EVM bytecode from Yul in the new via-IR pipeline - Daniel Kirchner
```
why not like that?

Reply:
harmonized capitalization in all titles ✔️